### PR TITLE
WFLY-13475 MP FT Quickstart: Change server startup documentation to reference local installation and standalone-microprofile.xml

### PR DESCRIPTION
**Pull request by rhusar**
WFLY-13475 MP FT Quickstart: Change server startup documentation to reference local installation and standalone-microprofile.xml

### CODE REVIEW

**Comment by rhusar** [comment]
@emmartins for review.

File: shared-doc/define-standalone-server-attributes.adoc [hunk 1]
```diff
@@ -37,6 +37,13 @@ ifeval::["{standalone-server-type}"=="ha"]
 :serverArguments:  -c standalone-ha.xml
 endif::[]
 
+// Start the standalone server with the Eclipse MicroProfile profile.
```
Comment (emmartins) [inline shared-doc/define-standalone-server-attributes.adoc:40]: minor enhancement, please add microprofile option in header of the file

Comment (rhusar) [reply shared-doc/define-standalone-server-attributes.adoc:41]: Done.

**Comment by emmartins** [comment]
other than minor enhancement requested, all good to me

**Pull request merged**